src/core/agent/services/ipexCommunicationService.ts

```diff
@@ -468,9 +468,30 @@ class IpexCommunicationService extends AgentService {
     }
   }
 
-  async getAcdcFromIpexGrant(said: string): Promise<ACDCDetails> {
+  async getAcdcFromIpexGrant(
+    said: string
+  ): Promise<Omit<ACDCDetails, "status">> {
     const exchange = await this.props.signifyClient.exchanges().get(said);
-    return exchange.exn.e.acdc;
+    const schemaSaid = exchange.exn.e.acdc.s;
+    const schema = await this.props.signifyClient.schemas().get(schemaSaid);
```

Comment:
What if the schema has not been resolved yet?

Reply:
Can you tell me when the schema has not been resolved or when I can reproduce that case?

Reply:
If this is the first time this wallet is receiving the issuance of schema X, `processExnIpexGrantNotification` won't do anything and we will create the notification record. The UI will then call this, and the schema will not have been resolved yet.

We might change it so that it calls `createLinkedIpexMessageRecord` with a new type of linked IPEX msg first (offering to issue a credential), and we can resolve there, but for now we don't have that. So I would resolve the schema here (and chained schemas) and add a todo that it may be unnecessary if we create an connection history item for offering to issue in the future.

To reproduce, easiest thing to do is to delete your KERIA docker volume and your DB in your browser and re-connect. If you don't want to do that, you can delete the DB in your browser AND the seed phrase (bran) in secure storage and re-onboard.